docs/client.rst

```diff
@@ -13,6 +13,28 @@ This will give you a ``tunneldigger`` directory which contains the broker and
 the client in separate directories. Client code can be compiled into a
 stand-alone program.
 
+In order to compile client code following packages are required on Ubuntu/Debian::
+
+    build-essential pkg-config cmake libnl-genl-3-dev libnl-3-dev libasyncns-dev
+    
+Root permission are needed for package installation and Tunneldigger client installation.
+
+Installation
+------------
+
+If we assume that you are currently in ``/tunneldigger/client`` folder you can do::
+
+    cmake -G "Unix Makefiles"
```

Comment:
the `-G ...` is unnecessary AFAIK, I have never seen it in build instructions.

Also, cmake prefers out-of-source builds. Why do you build in the source dir here?